docs/1.0-Installation.md

```diff
@@ -175,13 +176,13 @@ The operator also needs a ConfigMap that has:
     "kind": "ConfigMap",
     "metadata": {
         "name": "aws-account-operator-configmap",
-        "namespace": "aws-account-operator",
+        "namespace": "aws-account-operator"
     }
 }
 ```
 The ConfigMap could be generated and deployed with the `hack/scripts/set_operator_configmap.sh` script.
 
-    .hack/scripts/set_operator_configmap.sh -a ${ACCOUNT_LIMIT} -v ${VCPU_QUOTA} -r "${OSD_STAGING_1_OU_ROOT_ID}" -o "${OSD_STAGING_1_OU_BASE_ID}"
+    .hack/scripts/set_operator_configmap.sh -a ${ACCOUNT_LIMIT} -v ${VCPU_QUOTA} -r "${OSD_STAGING_1_OU_ROOT_ID}" -o "${OSD_STAGING_1_OU_BASE_ID}" -s ${AWS_ARN}
```

Comment:
Using the env vars that you've set in the README, wouldn't this be `-s ${STS_JUMP_ROLE_ARN}`